docs/editor/extension-marketplace.md

```diff
@@ -368,6 +368,16 @@ The Marketplace also provides you with resources to make an informed decision ab
 
 If you do see an extension that looks suspicious, you can report the extension to the Marketplace with the **Report Abuse** link at the bottom of the extension **More Info** section.
 
+### The extension signature cannot be verified by VS Code
+
+The Visual Studio Marketplace signs all extensions when they are published. VS Code verifies this signature when you install an extension to check the integrity and the source of the extension package.
+
+When you install an extension, you might see the following error message: `Cannot install extension because Visual Studio Code cannot verify the extension signature`.
+
+This error can be caused by a bad signature in the extension package, or by package integrity issues. Installing an extension with a bad signature can be unsafe.
```

Comment:
While this is all true, it is very unlikely that someone will encounter a bad signature or package integrity issues.

Far more likely, verification failures will be due to:

* partial file download
* unsupported scenarios (e.g.:  verifying under QEMU emulation)
* something else

However, it's not easy (currently) to diagnose the exact root problem.  Perhaps this would be a more accurate phrasing:

_This error can be caused by a variety of reasons, such as an unsupported environment or even (unlikely) package integrity issues.  Should you encounter this error, exercise caution before deciding to install anyway._